test/models/member_test.rb

```diff
@@ -175,4 +175,18 @@ class MemberTest < ActiveSupport::TestCase
 
     assert_nil member.last_membership
   end
+
+  test "schedules a job for updating Neon membership when enabled" do
+    member = create(:member)
+    mock = Minitest::Mock.new
+    mock.expect :call, true, [Integer]
+
+    NeonMemberJob.stub :perform_async, mock do
+      member.stub :can_update_neon_crm?, true do
+        member.update(city: "#{member.city} Test")
+      end
+    end
+
+    assert mock
```

Comment:
I think this needs to be `assert_mock`?

```suggestion
    assert_mock mock
```